README.md

```diff
@@ -16,38 +16,62 @@ pip install ansible-inventory-grapher
 Usage: ansible-inventory-grapher [options] pattern1 [pattern2...]
 
 Options:
+  --version             show program's version number and exit
   -h, --help            show this help message and exit
-  -i INVENTORY          specify inventory host file
-                        [/Users/will/.ansible/hosts]
+  -i INVENTORY          specify inventory host file [/etc/ansible/hosts]
   -d DIRECTORY          Location to output resulting files [current directory]
   -o FORMAT, --format=FORMAT
-                        python format string to name output files (e.g. {}.dot) 
-                        [defaults to stdout]
+                        python format string to name output files (e.g.
+                        {}.dot) [defaults to stdout]
   -q, --no-variables    Turn off variable display in default template
   -t TEMPLATE           path to jinja2 template used for creating output
   -T                    print default template
+  -a ATTRIBUTES         include top-level graphviz attributes from
+                        http://www.graphviz.org/doc/info/attrs.html
+                        [rankdir=TB;]
+  --ask-vault-pass      prompt for vault password
+  --vault-password-file=VAULT_PASSWORD_FILE
+                        Location of file with cleartext vault password
 ```
 
 Using the example inventory in https://github.com/willthames/ansible-ec2-example,
 we can generate the dot files for two of the example web servers using:
+```bash
+bin/ansible-inventory-grapher -i ../ansible-ec2-example/inventory/hosts \
+  prod-web-server-78a prod-web-server-28a \
+  -d test --format "test-{hostname}.dot"
 ```
-bin/ansible-inventory-grapher -i ../ansible-ec2-example/inventory/hosts prod-web-server-78a prod-web-server-28a -d test --format "test-{hostname}.dot"
+
+You can add the `-a` option to insert a string with graphviz attributes (http://www.graphviz.org/doc/info/attrs.html) to apply to the root level of the graph.  Some fun examples:
```

Comment:
Oh I thought you preferred leaving the full link, so you can see the URL without having to mouseover.  But if you'd like to convert everything over, we can do that in a later PR.